l10n_br_fiscal/models/res_partner.py

```diff
@@ -33,12 +33,26 @@ def _default_fiscal_profile_id(self, is_company=False):
         tracking=True,
     )
 
+    legal_nature_id = fields.Many2one(
+        comodel_name="l10n_br_fiscal.legal.nature",
+        string="Legal Nature",
+    )
+
     cnae_main_id = fields.Many2one(
         comodel_name="l10n_br_fiscal.cnae",
         domain=[("internal_type", "=", "normal")],
         string="Main CNAE",
     )
 
+    cnae_secondary_ids = fields.Many2many(
+        comodel_name="l10n_br_fiscal.cnae",
+        relation="res_partner_fiscal_cnae_rel",
+        column1="company_id",
+        column2="cnae_id",
+        domain="[('internal_type', '=', 'normal'), ('id', '!=', cnae_main_id)]",
+        string="Secondary CNAE",
```

Comment:
```suggestion
        string="Secondary CNAEs",
```
por ser um many2many acho que pode ser no plural aqui